include/arch/arc/v2/linker.ld

```diff
@@ -114,6 +114,9 @@ SECTIONS {
 		*(".rodata.*")
 		*(.gnu.linkonce.r.*)
 
+/* Located in generated directory. This file is populated by the
+ * add_linker_snippet() Cmake function. */
```

Comment:
I think you have to have the ending `*/` on a new line as discussed earlier.

Reply:
Done